Fix "by using" in concepts N--Z (#3207)
theforeman/foreman-documentation: guides/common/modules/con_troubleshooting-postgresql.adoc

# dnf module reset postgresql
----

> If a database was previously created by using PostgreSQL 10, perform an upgrade:

Reviewer: ```suggestion
If you created a PostgreSQL 10 database, perform an upgrade:
```
Just a suggestion to avoid passive language.
